src/ui/components/TestSuite/views/TestRecording/TestRecordingEvents/UserActionEventRow.tsx

```diff
@@ -142,11 +142,7 @@ export default memo(function UserActionEventRow({
       onMouseLeave={() => setIsHovered(false)}
     >
       <div className={styles.Text}>
-        {eventNumber != null ? (
-          <span className={styles.Number}>{eventNumber}</span>
-        ) : (
-          <span className={styles.ChainedEvent}>.</span>
```

Comment:
Can we also clean up the (now unused) `. ChainedEvent` class in `UserActionEventRow.module.css`?